### PR TITLE
Updating the BLISS in action section

### PR DESCRIPTION
**Pull request by SylLutze**
Feel free to add more to this, especially in the "lessons learned" section!

### CODE REVIEW

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 1]
```diff
@@ -330,10 +330,29 @@ results.
...
+
+Our team has been able to implement these BLISS guidelines in our own work. 
+Here we chronical guides, papers and presentations that include BLISS best 
+practices, as well as some lessons learned. 
+
+4.1 Examples of Our Work with BLISS
+-----------------------------------
+
+#. A `BLISS guide for Topics Concerning Pregnancy <https://uwnetid.sharepoint.com/:w:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B3E09642A-1A84-4D63-8690-EDA98E611295%7D&file=202204_Gender%20Neutral%20Language%20for%20Topics%20concerning%20Pregnancy.docx&action=default&mobileredirect=true>`_. This includes recommendations on gender inclusive language that can be used for topics of pregnancy and child birth. 
+#. A presentation about IV iron at the `Institute of Disease Modeling symposium <https://uwnetid.sharepoint.com/:p:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B141E4585-6A1B-414C-8DDE-2F075FEB1667%7D&file=20230501_IDM_Symposium_DRAFT_v0.2.pptx&action=edit&mobileredirect=true>`_. This presenation included many of the terms from out pregnancy guide implemented in practice.
```
Comment (aflaxman) [inline docs/source/model_design/general_reference_material/bliss/index.rst:342]:
```suggestion
#. A presentation about IV iron at the `Institute of Disease Modeling symposium <https://uwnetid.sharepoint.com/:p:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B141E4585-6A1B-414C-8DDE-2F075FEB1667%7D&file=20230501_IDM_Symposium_DRAFT_v0.2.pptx&action=edit&mobileredirect=true>`_. This presentation included many of the terms from out pregnancy guide implemented in practice.
```

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 2]
```diff
@@ -330,10 +330,29 @@ results.
 
 4.0 BLISS in Action 
 +++++++++++++++++++
-In this section, we will use case studies to explore the different ways in which 
-BLISS principles can be practically applied to specific areas of public health research. 
-Elaborate on lessons learned from other projects in which we've used BLISS (e.g., VEHSS
-DR paper; IV Iron communications and IDM Symposium)
+
+Our team has been able to implement these BLISS guidelines in our own work. 
+Here we chronical guides, papers and presentations that include BLISS best
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:335]:
```suggestion
Here we chronicle guides, papers and presentations that include BLISS best 
```

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 3]
```diff
@@ -330,10 +330,29 @@ results.
...
+
+Our team has been able to implement these BLISS guidelines in our own work. 
+Here we chronical guides, papers and presentations that include BLISS best 
+practices, as well as some lessons learned. 
+
+4.1 Examples of Our Work with BLISS
+-----------------------------------
+
+#. A `BLISS guide for Topics Concerning Pregnancy <https://uwnetid.sharepoint.com/:w:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B3E09642A-1A84-4D63-8690-EDA98E611295%7D&file=202204_Gender%20Neutral%20Language%20for%20Topics%20concerning%20Pregnancy.docx&action=default&mobileredirect=true>`_. This includes recommendations on gender inclusive language that can be used for topics of pregnancy and child birth. 
+#. A presentation about IV iron at the `Institute of Disease Modeling symposium <https://uwnetid.sharepoint.com/:p:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B141E4585-6A1B-414C-8DDE-2F075FEB1667%7D&file=20230501_IDM_Symposium_DRAFT_v0.2.pptx&action=edit&mobileredirect=true>`_. This presentation included many of the terms from out pregnancy guide implemented in practice.
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:342]:
```suggestion
#. A presentation about IV iron at the `Institute of Disease Modeling symposium <https://uwnetid.sharepoint.com/:p:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7B141E4585-6A1B-414C-8DDE-2F075FEB1667%7D&file=20230501_IDM_Symposium_DRAFT_v0.2.pptx&action=edit&mobileredirect=true>`_. This presentation included many of the terms from our pregnancy guide implemented in practice.
```

File: docs/source/model_design/general_reference_material/bliss/index.rst [hunk 4]
```diff
@@ -330,10 +330,29 @@ results.
...
+#. A paper on the `prevalence of diabetic retinopathy <https://jamanetwork.com/journals/jamaophthalmology/article-abstract/2806093>`_ that includes a "nondifferentiated sex and gender measure" for data anlysis and helpful explanation of what that means with respect to the specific primary data sources used.
```
Comment (zmbc) [inline docs/source/model_design/general_reference_material/bliss/index.rst:343]:
```suggestion
#. A paper on the `prevalence of diabetic retinopathy <https://jamanetwork.com/journals/jamaophthalmology/article-abstract/2806093>`_ that includes a "nondifferentiated sex and gender measure" for data analysis and helpful explanation of what that means with respect to the specific primary data sources used. 
```

**Review by zmbc — APPROVED** [review]
Nice! I agree with these lessons learned, and don't think I have any extra ones to add.

**Review by pletale — APPROVED** [review]
Looks great, thanks Syl!

**Pull request merged**